code/modules/power/supermatter/supermatter_effects.dm

```diff
@@ -0,0 +1,103 @@
+/datum/component/supermatter_glow
+/// The current light colour. Default is the nice yellow!
+	var/light_color = SUPERMATTER_COLOUR
+/// Typecasted reference to our supermatter(parent)
+	var/obj/machinery/power/supermatter_crystal/our_supermatter
+/// Just in case a bug gets introduced later on, let's not crash people!
+	var/loop_started = FALSE
```

Comment:
this is a very confusing comment and I'm a little unsure of what it means. Also while you're here could you indent all the comments on this declaration?